### PR TITLE
New "Deployment Settings" section for scale-down and hibernation schedule

### PR DESCRIPTION
**Pull request by paolaperaza**
🌞  @jwitz Good morning! I sent you a note in Slack, but this is a PR for the scale-down feature that you, me, and Conor discussed last week.

It turns out we need it for the platform release tomorrow morning at by 9:00am EST, so I put this PR together to make it as easy for you to finalize and get it live first thing in the morning. We may have to go back and edit again after we merge.

For now, a few notes for you:

- This feature is NOT in Dev yet and there is no way for you to access it directly, unfortunately.
- I created a new section in "Deployment Settings." I don't feel married to the H2 title, but we DO need to finalize this URL for marketing.
- I couldn't find a way to not have two H3s for creating the schedule and waking up the Deployment. There was so much intro content that I feel like we needed to cover.
- I included all the constraints and limitations and technical details that were available.

The new timeline is: `Dec 12` on cosmic energy for demos, `Dec 19` for customer availability. Dec 19th is Private Preview still.

My resources were these two Slack threads:

- [Latest Screenshots](https://astronomer.slack.com/archives/C063N37HQCT/p1701190164456779)
- [Recent conversation on UX and more screenshots](https://astronomer.slack.com/archives/C063N37HQCT/p1700597239301569)

As for what's needed:

- I did NOT end up uploading screenshots. Could you take care of that where needed?
- Take a look at everything and feel free to modify any/all in-line. I don't necessarily need to review this again.
- Feel free to merge this PR as soon as you think it's good enough. The sooner I think the better, and we can iterate after.

### CODE REVIEW

File: astro/deployment-settings.md [hunk 1]
```diff
@@ -244,6 +244,61 @@ Every Deployment has two PgBouncer Pods assigned to two different nodes to preve
 
 :::
 
+### Hibernate a Deployment
+
+:::caution
+
+This feature is in [Private Preview](feature-previews.md). To access this feature or learn more, contact [Astronomer Support](https://cloud.astronomer.io/support). All customers are eligible to use this feature upon request.
```
Comment (conorbev) [inline astro/deployment-settings.md:251]: Product page is linking the contact us here: [Please reach out](https://www.astronomer.io/contact/?__hstc=76629258.92c76fbe32d5d0b999770b541958c1d9.1701843969589.1701843969589.1701843969589.1&__hssc=76629258.1.1701843969590&__hsfp=2556139473) Any reason not to use the same?

File: astro/deployment-settings.md [hunk 2]
```diff
@@ -244,6 +244,61 @@ Every Deployment has two PgBouncer Pods assigned to two different nodes to preve
...
+
+For example, you might need to test a DAG during working hours on Monday and Tuesday, but you promote that DAG to production at the end of your testing and do not need to use your development environment for any other testing until the following week. In this scenario, you would create a hibernation schedule with a start time of Wednesday at 9am and an end time of Monday at 9am. During this time, your Deployment settings are preserved and your cost on Astro for the Deployment is $0. When the hibernation schedule ends, you can resume using the Deployment.
```
Comment (conorbev) [inline astro/deployment-settings.md:257]: I think this is fine for now, but, the use case here reads more to me like the "on-demand" hibernation style where I wouldn't set an explicit schedule but would just hibernate it when I'm done and resume it the following week as needed.

**Review by conorbev — COMMENTED** [review]
We should probably update the Contact us link since I don't think support is prepped to handle any inbound requests to be part of the private preview vs Vivian who will receive any msgs at: https://www.astronomer.io/contact/?__hstc=76629258.92c76fbe32d5d0b999770b541958c1d9.1701843969589.1701843969589.1701843969589.1&__hssc=76629258.1.1701843969590&__hsfp=2556139473 _is_

**Review by ianbuss — COMMENTED** [review]
Had a pending review, sorry thought I'd submitted it @jwitz

**Pull request merged**